makefile

```diff
@@ -3,7 +3,6 @@ AL_LIBS = $(shell allegro-config --libs)
 CFLAGS ?= -O2 -Wall
 
 ifdef DEBUGMODE
```

Comment:
Actually I mean the entire DEBUGMODE build option.

And while we're at it, RELEASE is silly too. I don't either -O3 nor -s. :P

Reply:
Yes, that's right, I noted that too but did not change it assuming that -O2 vs. -O3 had some reason to be used. Notice that -O2 was part of the initial code. What I did instead was simply remove the symbols from the release compilation, confining them to the DEBUGMODE flag. For other improvements, I would recommend a step-by-step approach also assessed on the free time available for the implementation.